plugins/modules/network_settings_intent.py

```diff
@@ -785,15 +785,16 @@ def get_network_params(self, site_id):
                 },
                 "netflowcollector": {
                     "ipAddress": netflow_details.get("value")[0].get("ipAddress"),
-                    "port": netflow_details.get("value")[0].get("port"),
-                    "configureDnacIP": netflow_details.get("value")[0].get("configureDnacIP"),
+                    "port": netflow_details.get("value")[0].get("port")
                 },
                 "timezone": timezone_details.get("value")[0],
             }
         }
         network_settings = network_details.get("settings")
-        if dhcp_details is not None:
+        if dhcp_details.get("value") != []:
```

Comment:
As we already got the value of dhcp_details by using "dhcp_details.get("value")", do we need to get again in next line? Can we optimize it by storing and reusing it..

```
dhcp_value = dhcp_details.get("value")
if dhcp_value:
    network_settings.update({"dhcpServer": dhcp_value})
else:
    network_settings.update({"dhcpServer": [""]})
```
(or)
As we call the same API "network_settings.update" in if and else, we can write below:
```
dhcp_value = dhcp_details.get("value")
network_settings.update({"dhcpServer": dhcp_value if dhcp_value else [""]})
```